packages/@aws-cdk/aws-rds/README.md

```diff
@@ -613,4 +610,7 @@ cluster.grantDataApiAccess(fn);
 
 **Note**: To invoke the Data API, the resource will need to read the secret associated with the cluster.
 
+**Note**: Using the Data API, you can interact with a ServerlessCluster without using its VPC. Therefore, the parameter "vpc" is optional. 
+The cluster will be created in a VPC, but you will know nothing about it.
+
```

Comment:
Actually, let's leave this example as-is, and add a new sub-section, with the appropriate header level, talking about VPC being optional.

If optional VPC and `enableDataApi` are indeed separate, let's separate them in our documentation as well.